setup.py

```diff
@@ -42,7 +42,7 @@
     "ffmpeg",
     "cftime",
     "netcdf4",
-    "dask",
+    "dask<=2020.12.0",
```

Comment:
Ideally we'd fix the regression in a separate PR, rather than pin this here.

Reply:
Sure, I'll make a quick PR, revert this and can rebase once merged.

Reply:
Addressed in a293d626. New PR is https://github.com/holoviz/holoviews/pull/4803